### PR TITLE
feat: fetch raw enums from bff basic concept

### PR DESCRIPTION
**Pull request by assafchamoy**

### CODE REVIEW

File: src/App.tsx [hunk 1]
```diff
@@ -86,8 +89,10 @@ const App: React.FC = () => {
       >
         <RMWCThemeProvider className={`app-container ${theme.type}-theme`} options={theme}>
           <CssBaseline />
-          <StaticDataFetcher />
-          <DiscreteLayerView />
+          <EnumsMapContext.Provider value={{ enumsMap, setEnumsMap }}>
```
Comment (alebinson) [inline src/App.tsx:92]: setEnumsMap used from <MCEnums> which changes outer context value... less preferable...but it happens only once .

File: src/common/contexts/enumsMap.context.ts [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+export interface IEnumsMapType {
+  [unionEnumKey: string]: IEnumDescriptor;
+}
+interface IEnumsMapContext {
+  enumsMap: IEnumsMapType | null;
+  setEnumsMap: (enums: IEnumsMapType) => void
+}
+
+export const DEFAULT_ENUM_DESCRIPTOR: IEnumDescriptor = {
+    enumName: 'MISSING ICON',
```
Comment (alebinson) [inline src/common/contexts/enumsMap.context.ts:21]: *##UNKNOWN_ENUM_TYPE##*

File: src/common/contexts/enumsMap.context.ts [hunk 3]
```diff
@@ -0,0 +1,36 @@
...
+    icon: 'mc-icon-Close glow-missing-icon',
+    parent: '',
+    properties: {},
+    realValue: 'Missing Value',
```
Comment (alebinson) [inline src/common/contexts/enumsMap.context.ts:25]: *##MISSING_VALUE##*

**Review by alebinson — CHANGES_REQUESTED** [review]
see my comments

File: src/App.tsx (hunk 1 above)
Comment (assafchamoy) [reply src/App.tsx:92]: Not very clear.. could you rephrase please?

Comment (alebinson) [reply src/App.tsx:92]: You change the context value from inner/nested component basically you should only read context value in nested components

**Review by alebinson — APPROVED** [review]
